ntoskrnl/ex/dbgctrl.c

```diff
@@ -155,9 +155,9 @@ ExpDebuggerWorker(
  * abilities (if needed to at all)
  *
  * See: http://www.osronline.com/showthread.cfm?link=93915
- *      http://void.ru/files/Ntexapi.h
- *      http://www.codeguru.com/code/legacy/system/ntexapi.zip
- *      http://www.securityfocus.com/bid/9694
+ *      https://web.archive.org/web/20110703100145/http://void.ru/files/Ntexapi.h
+ *      https://web.archive.org/web/20240527134904/https://www.codeguru.com/code/legacy/system/ntexapi.zip
+ *      http://www.securityfocus.com/bid/9694 (DEAD_LINK)
```

Comment:
As you can see in https://github.com/reactos/reactos/pull/7426/files , these links are going away because they actually didn't bring much value.

Reply:
Reverted.